### PR TITLE
Manual trigger integration tests

### PR DESCRIPTION
**Pull request by roasher**
To be able to move with https://github.com/finos/legend-engine/pull/809 there is a need to be able to run integreation tests manually. This should also fix [Database BigQuery Integration Tests](https://github.com/finos/legend-engine/actions/runs/2871956616) runs

### CODE REVIEW

**Comment by epsstan** [comment]
Enabling manual runs allows any contributor to run tests against our test databases.  Not sure about the compute and other costs of doing so.

**Comment by roasher** [comment]
> Enabling manual runs allows any contributor to run tests against our test databases. Not sure about the compute and other costs of doing so.

According to discussion below one need to have write permissions to do so.
> To trigger a workflow in a repository, the user should be a collaborator with Write permission in the repository. Normally the external users can’t trigger workflows in the repository.
https://github.com/orgs/community/discussions/26053

File: .github/workflows/database-bigquery-integration-test.yml [hunk 1]
```diff
@@ -69,4 +70,4 @@ jobs:
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_INTEGRATION_USER1_SECRET_ACCESS_KEY }}
         run: |
           mvn -P run-bigquery-integration -pl legend-engine-xt-relationalStore-executionPlan-connection-tests -Dtest="ExternalIntegration*BigQuery" test
-          mvn -P run-bigquery-integration -pl legend-engine-xt-relationalStore-executionPlan-connection-authentication -Dtest="ExternalIntegration*BigQuery*" test
+          mvn -P run-bigquery-integration -pl legend-engine-xt-relationalStore-executionPlan-connection-authentication-default -Dtest="ExternalIntegration*BigQuery*" test
```
Comment (epsstan) [inline .github/workflows/database-bigquery-integration-test.yml:73]: Good catch!

File: .github/workflows/database-spanner-integration-test.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+name: Database Spanner Integration Tests
+
+env:
+  CI_DEPLOY_USERNAME: ${{ secrets.CI_DEPLOY_USERNAME }}
+  CI_DEPLOY_PASSWORD: ${{ secrets.CI_DEPLOY_PASSWORD }}
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+
+on:
```
Comment (epsstan) [inline .github/workflows/database-spanner-integration-test.yml:23]: We don't want Spanner test to be manually triggered ?

**Review by epsstan — APPROVED** [review]

File: .github/workflows/database-spanner-integration-test.yml (hunk 2 above)
Comment (roasher) [reply .github/workflows/database-spanner-integration-test.yml:23]: We do. Missed that. Fixed.

File: .github/workflows/database-bigquery-integration-test.yml (hunk 1 above)
Comment (roasher) [reply .github/workflows/database-bigquery-integration-test.yml:73]: You are welcome :)

**Comment by finos-admin** [comment]
This PR is stale because it has been open for 30 days with no activity. Please remove stale label or add any comment to keep this open. Otherwise this will be closed in 5 days.

**Comment by roasher** [comment]
@vikask-gs do you need this?

**Comment by vikask-gs** [comment]
> @vikask-gs do you need this?

I think we can close this, as we have other PRs which cover these changes

**Pull request closed without merging**